draft-beky-httpbis-metadata.md

```diff
@@ -110,12 +113,21 @@ that receives METADATA for a stream in the “idle” state _MAY_ choose to reta
 the payload for a period of time, under the assumption that the stream will soon
 transition to the “open” state.
 
+A metadata block is the concatenation of the payloads of a sequence of one or
+more METADATA frames, among which only the last one carries the END_METADATA
+flag.   If the transfer of the last metadata block cannot be completed due to
```

Comment:
optional:

"..., among which only the last one carries the END_METADATA flag."

-->

"..., the last of which has the END_METADATA flag set."

Reply:
Done, except I added "_only_ the last of which" to make it explicit that the rest of the frames must not have the flag set.  This might still has room for improvement, let me know if you have any further suggestions.